server/enterprise/message_export/global_relay_export/global_relay_export.go

```diff
@@ -31,22 +31,28 @@ const (
 	GlobalRelayChannelNameHeader = "X-Mattermost-ChannelName"
 	GlobalRelayChannelIDHeader   = "X-Mattermost-ChannelID"
 	GlobalRelayChannelTypeHeader = "X-Mattermost-ChannelType"
-	MaxEmailBytes                = 250 << (10 * 2)
 	MaxEmailsPerConnection       = 400
 )
 
+// MaxEmailBytes is a var because it needs to be set in tests. Otherwise it shouldn't be touched.
+var MaxEmailBytes int64 = 250 << (10 * 2)
```

Comment:
Should we simplify to `250 * 1024 * 1024` since we are here? It's easier to understand we are talking about 250MBs :)

Reply:
Yep, much better, thanks.